api/swagger.yaml

```diff
@@ -153,8 +153,17 @@ definitions:
       id:
         type: "integer"
         format: "int64"
+      brickLength:
+        type: "string"
```

Comment:
Please change to Integer. Values: 20,40,60

Reply:
@lindsaymacvean , done, please review next commit.